modules/admin/client/components/AdminMessages.component.js

```diff
@@ -10,120 +10,121 @@ import UserLink from './UserLink.component';
 // Mongo ObjectId is always 24 chars long
 const MONGO_OBJECT_ID_LENGTH = 24;
 
-export default class AdminMessages extends Component {
-  constructor(props) {
-    super(props);
-    this.onUserChange = this.onUserChange.bind(this);
-    this.onSubmit = this.onSubmit.bind(this);
-    this.state = {
-      messages: [],
-      user1: '',
-      user2: '',
-    };
-  }
+export default function AdminMessages() {
```

Comment:
Sorry, I refactored this component while at it — I should PR this separately to make this one easier to review...

Reply:
IMO it's fine to leave it for reviews as it is. Too much hassle to separate the two now...